src/test/utils.js

```diff
@@ -15,6 +15,11 @@ export const withUserContext = (Component, user, route) => {
   );
 };
 
-export const withTheme = Component => (
-  <ThemeProvider>{Component}</ThemeProvider>
-);
+export const withTheme = (Component, route) => {
+  if (route) history.push(route);
```

Comment:
Doesn't this add too much responsibility to this simple util function?

Reply:
moved to withHistory hoc